packages/mobile/src/config.ts

```diff
@@ -79,3 +79,8 @@ export const MOONPAY_PUBLIC_KEY = keyOrUndefined(
   'MOONPAY_PUBLIC_KEY'
 )
 export const MOONPAY_RATE_API = `https://api.moonpay.io/v3/currencies/celo/price?apiKey=${MOONPAY_PUBLIC_KEY}`
+
+export const GETH_USE_FULL_NODE_DISCOVERY = true
```

Comment:
For env specific variables (for example, if this would not work on `alfajores` but would on `mainnet`), we use the mobile .env , .env.mainnet, .env.alfajores files for this configuration: https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/.env

For more feature flag type variables that are constant across networks, we use: https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/flags.ts

Would these flags fit in one of those files instead? I think .env is most natural as that's where other geth-related flags like `DEFAULT_SYNC_MODE` and `FORNO_ENABLED_INITIALLY` live